zipline/pipeline/graph.py

```diff
@@ -16,6 +15,14 @@ class CyclicDependency(Exception):
     pass
 
 
+# This sentinel value is uniquely-generated at import time so that we can
+# guarantee that it never conflicts with a user-provided column name.
+#
+# (Yes, technically, a user can import this file and pass this as the name of a
```

Comment:
can we make this impossible?

Reply:
I don't think there's any plausible scenario in which this will matter unless a user is intentionally trying to break something, in which case I don't feel bad if this breaks.